### DISCUSSION TITLE
Automatically add timetamp as name when creating a new empty page

### DISCUSSION THREAD

**Discussion by bruvellu** [zim-desktop-wiki/zim-desktop-wiki#2656] [Ideas]
Hey, I'm using [Markor](https://github.com/gsantner/markor/) to create and edit Zim notes on my mobile, and it has a feature that I find really useful. I can create a new note without giving it a title, and it'll automatically create a page using a timestamp as the page name and title. With two clicks, I can start writing without the need to think of a title. It's great for quick notes or journaling.

For example, creating an empty note in Markor will generate the file `2024-08-20T092435.txt` with these contents:

```
Content-Type: text/x-zim-wiki
Wiki-Format: zim 0.6
Creation-Date: 2024-08-20T09:24:35+02:00

====== 2024-08-20T092435 ======
Created Tuesday 20 August 2024
```

As far as I know, doing this in Zim isn't possible, since you can't create a page without a title using the New Page dialog. It can be done with a custom tool, of course, but perhaps this is a functionality that might be useful for more people as a plugin or even core? Has anyone implemented something similar? Would love to hear ideas.

**Comment by jaap-karssenberg** [comment]
Please have a look at the "quick note" plugin, might not be perfect, but it
is the palce for this kind of use case in zim

**Reply by bruvellu** [reply]
Hi Jaap, thanks for the pointer. That's indeed close to what I wanted. I gave it a try with a custom template and ran into one issue, which I solved in this new PR #2660. Take a look and see what you think
